crates/pipeline_manager/src/main.rs

```diff
@@ -1624,8 +1626,15 @@ async fn http_input(
     ),
     params(
         ("pipeline_id" = Uuid, Path, description = "Unique pipeline identifier."),
-        ("table_name" = String, Path, description = "SQL table name."),
+        ("table_name" = String, Path, description = "SQL table or view name."),
```

Comment:
Nit: should we say "relation" when something can be a table or a view?

Reply:
It might be good to be explicit here. A program consists of `CREATE TABLE` and `CREATE VIEW` declarations; I wanted to make it clear that both can be queried.

Reply:
@ryzhyk yes, and that means the parameter shouldn't be called `table_name` because views can be queried too.